deploy/infra/aws/terraform/cloud-native/cluster-infra/output.tf

```diff
@@ -67,3 +77,33 @@ output "rds_port" {
   description = "The port on which the RDS instance accepts connections"
   value       = try(module.cp_rds.db_instance_port, null)
 }
+
+output "cp_efs_filesystem_id" {
+  description = "ID of the created efs filesystem"
+  value = try(module.cp_system_efs[0].id, null)
+}
+
+output "cp_efs_filesystem_exec_role" {
+  description = "Execution role with permission to interact with efs filesystem, used by SCI driver"
+  value = try(module.efs_csi_irsa[0].iam_role_arn, null)
+}
+
+output "cp_fsx_filesystem_id" {
+  description = "ID of the created fsx filesystem"
+  value = try(aws_fsx_lustre_file_system.fsx[0].id, null)
+}
+
+output "cp_fsx_filesystem_exec_role" {
+  description = "Execution role with permission to interact with fsx filesystem, used by SCI driver"
+  value = try(module.fsx_csi_irsa[0].iam_role_arn, null)
+}
+
+output "cp_kms_arn" {
+  description = "ARN of created KMS key"
+  value = try(module.kms.key_arn, null)
+}
+
+output "eks_node_sg" {
+  description = "Internal security group id"
```

Comment:
the name and the description doesn't describe the resource well. rename it to `eks_cluster_primary_security_group_id` and change the desription